cmd/openshift-install/testdata/agent/image/validations/sno_invalid_workers.txt

```diff
@@ -1,40 +0,0 @@
-! exec openshift-install agent create image --dir $WORK
-
-stderr 'Total number of Compute.Replicas must be 0 for none platform. Found 1'
```

Comment:
Having master replicas = 1 and compute replicas = 1 is an invalid configuration. I think the validation should still heck this. Perhaps the error or validation message should be changed.